tests/unit/test_simulation.py

```diff
@@ -21,21 +21,26 @@ def test_simple_model(self):
 
     def test_basic_ops(self):
         model = pybamm.lithium_ion.SPM()
+        if model is None:
+            pytest.skip("Model is None, skipping test.")
+
```

Comment:
I'm not sure why this was needed. We're setting up the SPM just above this line, aren't we?